Havit.Blazor.Components.Web.Bootstrap/Forms/Internal/HxMultiSelectInternal.razor

```diff
@@ -5,8 +5,7 @@
 @{
 	bool enabled = EnabledEffective && (ItemsToRender != null);
 }
-<div class="@CssClassHelper.Combine("hx-multi-select",
-                                    HasInputGroups ? "input-group" : null,
+<div class="@CssClassHelper.Combine(HasInputGroups ? "input-group" : null,
```

Comment:
Why did you move the `.hx-multi-select` class to the inner `div`?
According to our conventions, the "component class" should be applied to the most outer HTML element to allow scoping of any additional CSS rules.

Reply:
Thanks it needed to have a correct display of the floating label if the class is not in the inner div it displays incorrect as Date component shows also incorrect when using Floating label (btw, I'm testing a fix for DatePicker component using floating labels)

But maybe we could find a talkaround, that's the one that worked for me, I'm using it in my production environment right now

Reply:
Just double-checked and the `hx-multi-select` class isn't used by any existing CSS rules in the library, so moving the class to the inner `div` won't have any effect. We want this class applied to the outermost HTML element to allow scoping for any custom CSS rules the user creates in their project.

Reply:
In my tests it shows the same as the HxInputDate shows look at the attached screenshot:
![image](https://github.com/user-attachments/assets/68ff695f-99b7-4255-88d2-f69f17d565d2)

Reply:
`HxInputDate` issue extracted to #951